front50-core/src/main/groovy/com/netflix/spinnaker/front50/model/StorageServiceSupport.java

```diff
@@ -219,6 +219,8 @@ public T findById(String id) throws NotFoundException {
   }
 
   public void update(String id, T item) {
+    item.setLastModifiedBy(AuthenticatedRequest.getSpinnakerUser().orElse("anonymous"));
+    item.setLastModified(System.currentTimeMillis());
```

Comment:
The object store backed services currently override the `lastModified` at load time but it's confusing if you're ever looking at the raw records outside of `front50`.